int32 obj_class
float32 probability
int32 x1
int32 x2
int32 y1
int32 y2
